<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Documents</class>
 <widget class="QWidget" name="Documents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>855</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1320</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>-1</y>
     <width>1021</width>
     <height>870</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>2222222</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/* В QSS-файле или строке */
QScrollArea {
    border: none;
    background: transparent;
}

/* Вертикальная полоса прокрутки */
QScrollBar:vertical {
    width: 10px;
    background: #2c2c2c;
}

QScrollBar::handle:vertical {
    background: #606060;
    border-radius: 5px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none;
}

/* Горизонтальная полоса прокрутки */
QScrollBar:horizontal {
    height: 10px;
    background: #2c2c2c;
}

QScrollBar::handle:horizontal {
    background: #606060;
    border-radius: 5px;
}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>870</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>730</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Создать новый документ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Создать новый документ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 50%;
background-color: rgba(0, 85, 255, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="NameButton">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>90</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgba(0, 85, 255, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
  </widget>
  <widget class="QTextEdit" name="nameTextEdit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>90</y>
     <width>281</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>45</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color: rgba(0, 85, 255, 55);
padding-left:10px</string>
   </property>
   <property name="placeholderText">
    <string>Имя пользоватиля</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FileButton">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>160</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgba(0, 85, 255, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
  </widget>
  <widget class="QTextEdit" name="FileTextEdit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>160</y>
     <width>281</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;;;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color: rgba(0, 85, 255, 55);
padding-left:10px</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:14pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>Название файла</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>230</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color: rgba(0, 85, 255, 55);
padding-left:10px</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DataButton">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>230</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgba(0, 85, 255, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>300</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color: rgba(0, 85, 255, 55);
padding-left:10px</string>
   </property>
   <item>
    <property name="text">
     <string>все форматы</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.doxc</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.pdf</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.xlsx</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.pptx</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.txt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.csv</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.odt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.ods</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.odp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.png</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 26px;
background-color: rgba(0, 85, 255, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
